test/libsolidity/lsp/publish_diagnostics_1.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+contract MyContract
+{
+    constructor()
+    {
+        uint unused; // [Warning 2072] Unused local variable.
+    }
+}
+
+contract D
+{
+    function main() public payable returns (uint)
+    {
+        MyContract c;// = new MyContract();
```

Comment:
my source code ideas for testing aren't world-class. I've changed this line. - All I wanted was to generate a warning. :-)

Reply:
Oh, I thought it was a piece of dead code :)

In that case it would be fine, but you should add a short comment at least hinting at what's happening here.